src/sentry/api/endpoints/auth_index.py

```diff
@@ -100,8 +112,21 @@ def put(self, request: Request):
             return Response(status=status.HTTP_401_UNAUTHORIZED)
 
         validator = AuthVerifyValidator(data=request.data)
-        if not validator.is_valid():
-            return self.respond(validator.errors, status=status.HTTP_400_BAD_REQUEST)
+        has_valid_sso_session = has_completed_sso(request, request.superuser.org_id)
+        from_superuser_modal = request.data.get("isSuperuserModal")
+
+        if not validator.is_valid() and not has_valid_sso_session:
+            try:
+                auth_identity = AuthIdentity.objects.get(user=request.user)
```

Comment:
what if the user has multiple AuthIdentitys?

Reply:
this would be the case if the user has an authidentity in multiple orgs right? i want to redirect them to the sso login of the org they're currently in, ill look for another way to do this